src/components/MenuPanel/MenuPanel.js

```diff
@@ -66,21 +67,51 @@ class MenuPanel extends React.PureComponent {
     connected: PropTypes.bool.isRequired,
     daoAddress: DaoAddressType.isRequired,
     notifications: PropTypes.number,
-    onOpenApp: PropTypes.func.isRequired,
     onNotificationClicked: PropTypes.func.isRequired,
+    onOpenApp: PropTypes.func.isRequired,
     onRequestAppsReload: PropTypes.func.isRequired,
+    viewportHeight: PropTypes.number,
   }
 
+  _animateTimer = -1
+  _contentRef = React.createRef()
+  _innerContentRef = React.createRef()
+
   state = {
     notifications: [],
     systemAppsOpened: systemAppsOpenedState.isOpen(),
     animate: false,
+    scrollVisible: false,
   }
 
   componentDidMount() {
-    setTimeout(() => this.setState({ animate: true }), 0)
+    this._animateTimer = setTimeout(() => this.setState({ animate: true }), 0)
```

Comment:
We've been doing this often, maybe we need to send a PR to `react-spring` with a new _don't_animate_on_first_render_prop_ PR.

Reply:
I was just adding the `clearTimeout()` because it was missing, but I think there are ways to achieve what we want without using `setTimeout` already. I’ll investigate 🕵️‍♂️